src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CommencementDateController.cs

```diff
@@ -22,24 +23,20 @@ public sealed class CommencementDateController : ControllerBase
     {
         private readonly ApplicationDbContext context;
 
-        public CommencementDateController(ApplicationDbContext context)
+        private readonly IOrderService orderService;
+
+        public CommencementDateController(ApplicationDbContext context, IOrderService orderService)
```

Comment:
We shouldn't need to inject `ApplicationDbContext` any more.

Reply:
Removed `ApplicationDbContext` injection